### PR TITLE
removed_pfodGUIdesigner_library

### PR DESCRIPTION
**Pull request by drmpf**
This library has been superceded by the free Android app [pfodGUIdesigner](https://play.google.com/store/apps/details?id=au.com.forward.pfodGUIdesigner).  Download that app from Google play and use it instead of this code. The pfodGUIdesigner app supports generating code for more boards and has the latest bug fixes. The code in this library has a few bugs which will not be fixed. The code is left here for reference only. To compile this code you need the SafeString library (V4.1.24) and the the pfodParser library V3.58.0.  The later versions of pfodParser library will not compile with this code.

### CODE REVIEW

**Comment by per1234** [comment]
Hello @drmpf. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by umbynos — APPROVED** [review]

**Comment by umbynos** [comment]
Done:
```
2022/12/12 16:56:55 Loaded 5386 libraries from DB
Removing pfodGUIdesigner
Success!
```

**Pull request merged**
